script/libtest.bash

```diff
@@ -284,4 +288,27 @@ test_kubeless_function_update() {
     update_function $func
     verify_update_function $func
 }
+test_kubeless_ingress() {
+    local func=${1:?} domain=example.com act_ingress exp_ingress
+    echo_info "TEST: ingress ${func}"
+    kubeless ingress create ing-${func} --function ${func} --hostname ${func}.${domain}
+    kubeless ingress list | fgrep -w ing-${func}
+    act_ingress=$(kubectl get ingress ing-${func} -ojsonpath='{range .spec.rules[*]}{@.host}:{@.http.paths[*].backend.serviceName}')
+    exp_ingress="${func}.${domain}:${func}"
+    [[ ${act_ingress} == ${exp_ingress} ]]
+    kubeless ingress delete ing-${func}
+}
+test_kubeless_autoscale() {
+    local func=${1:?} exp_autoscale act_autoscale
+    # Use some fixed values
+    local val=10 num=3
+    echo_info "TEST: autoscale ${func}"
+    kubeless autoscale create ${func} --value ${val:?} --min ${num:?} --max ${num:?}
+    kubeless autoscale list | fgrep -w ${func}
```

Comment:
ditto, it would be nice to drive some load (or whatever) and actually see the pod get autoscaled.

Reply:
... +1 == +2
fwiw 1st take on these, can work on improving it

Reply:
aha, `--min=3`.  I looked for that and didn't see it earlier.  lgtm :)